src/test/app/Escrow_test.cpp

```diff
@@ -4328,6 +4328,11 @@ struct Escrow_test : public beast::unit_test::suite
         testMetaAndOwnership(features);
         testConsequences(features);
         testEscrowWithTickets(features);
+    }
+
+    void
+    testXLS34WithFeats(FeatureBitset features)
```

Comment:
i was thinking of something like `testIOUSupportWithFeats` instead of the amendment number. same for paychan